python/xgbserver/xgbserver/model.py

```diff
@@ -33,13 +34,13 @@ def __init__(self, name: str, model_dir: str, booster: XGBModel = None):
     def load(self):
         model_file = os.path.join(
             kfserving.Storage.download(self.model_dir), BOOSTER_FILE)
-        self._booster = xgb.Booster(model_file=model_file)
+        self._booster = xgb.Booster(params={"nthread" : DEFAULT_NTHREAD}, model_file=model_file)
```

Comment:
How do you feel about wiring this through the API as a cli arg. The user can set the value on the XGBoostSpec and the kfserving controller will default it to whatever the resource requests are (i.e., if 3.6 cpu, then this value is 4)

Reply:
Ok I'll have a look on implementing this

Reply:
@ellis-bigelow I've updated the PR with your suggestion